Docs/Configuration.tex

```diff
@@ -2487,7 +2487,7 @@ \subsection{Quirks Properties}\label{kernelpropsquirks}
   \textbf{Description}: Increases 32-bit PCI bar size in IOPCIFamily from 1 to 4 GBs.
 
   \emph{Note}: This option should be avoided whenever possible. In general the necessity
-  of this option means misconfigured or broken firmware.
+  of this option means misconfigured or faulty firmware.
 
```

Comment:
'faulty' sounds too close  to gone wrong since purchase (&/or in many instances of the same thing it works, but in yours it doesn't); I suspect 'broken' (as informal technical usage for incorrectly programmed/designed) is more accurate to what was meant.